### DISCUSSION TITLE
Expanding the CustomVariable

### DISCUSSION THREAD

**Discussion by DrMarkX** [Snow-Shell/servicenow-powershell#204] [Q&A] [answered]
I like the new IncludeCustomVariable format. This makes it easier to display the minute data within that variable.  However, I used to be able to see all the details of the CustomVariable by using this one-liner.
`Get-ServiceNowRecord RITM888888 -IncludeCustomVariable | Select-Object CustomVariable -ExpandProperty CustomVariable`
That no longer expands all parts. Is there another way to get these data?

Old way shows data like this.
```
Name          : something_check
Value         : false
DispalayName  : Do you need this thing?
Type          : CheckBox
```

New Way shows data like this.
`something_chcck       : {Type, DisplayName, Value}`

**Comment by gdbarron** [comment, marked as answer]
Resolved with #206
